### DISCUSSION TITLE
i3 Rounded Corners

### DISCUSSION THREAD

**Discussion by AlexvZyl** [i3/i3#5374] [Q&A] [answered]
I am unable to find any information regarding rounded corners in the issues or the discussions.  If I missed it, please point me in the right direction.

What is the project team's stance on adding this feature?  I am feeling bold since i3-gaps was recently merged :) 

P.S. I did find https://github.com/Airblader/i3/issues/167, but that is from a fork.

**Comment by okraits** [comment, marked as answer]
@Airblader already wrote some statements regarding this in the issue you linked to. I'm quite sure there are no real plans to add something like this.

**Comment by neg-serg** [comment]
You can try to do it via picom with rounded corners support.

**Reply by AlexvZyl** [reply]
I am using picom, but i3 then does not round the borders correctly.
![image](https://user-images.githubusercontent.com/81622310/224023853-ebdd5ea9-a805-4a4b-a370-d4458703c2fd.png)

**Reply by okraits** [reply]
i3 doesn't round the borders at all :grin:

**Reply by AlexvZyl** [reply]
That is what I meant.  Used the wrong words :)
